ethcore/src/types/transaction.rs

```diff
@@ -390,6 +390,34 @@ impl Deref for LocalizedTransaction {
 	}
 }
 
+/// Queued information with additional information.
+#[derive(Debug, Clone, PartialEq, Eq, Binary)]
+pub struct PendingTransaction {
```

Comment:
You could implement `Deref<Target=SignedTransaction>` to get rid of some `tx.transaction.<functioncall>` statements.